src/defaultsettings.cpp

```diff
@@ -174,6 +174,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("leaves_style", "fancy");
 	settings->setDefault("connected_glass", "false");
 	settings->setDefault("smooth_lighting", "true");
+	settings->setDefault("simplified_smooth_lighting", "false");
```

Comment:
Default should be 'true', as in settingtypes.txt.

Reply:
Fixed